db/migrate/20160310044144_add_doc_id_to_topic.rb

```diff
@@ -0,0 +1,26 @@
+class AddDocIdToTopic < ActiveRecord::Migration
+
+  def up
+    add_column :topics, :doc_id, :integer, default: 0
+    add_column :docs, :topics_count, :integer, default: 0
+    add_column :docs, :allow_comments, :boolean, default: true
+
+    if Forum.where(name: 'Doc comments').first.nil?
```

Comment:
I think there needs to be a forum for comments on the documents, which is what this is.  I did add it to the 'seeds.rb` but for people upgrading, we have to ensure the forum gets added.  Any other ideas on where to put it?